client/components/Catalog.jsx

```diff
@@ -2,49 +2,53 @@ import React, { useState, useEffect } from 'react';
 import axios from 'axios';
 import { Navbar } from './Navbar';
 import { Carasouel } from './Carasouel';
+import { Link } from 'react-router-dom';
 
 export const Catalog = () => {
   const [Sdata, setData] = useState([]);
-  // const [error, setError] = useState(null);
-  // const [isLoaded, setIsLoaded] = useState(false);
-  // const [choosenItem, setChoosenItem] = useState(null);
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
   const catalogItems = () => {
     const options = {
       method: 'GET',
-      url: 'http://localhost:3000/products'
+      url: 'http://localhost:3000/catalog'
     };
     axios.request(options).then(response => {
       setData(response.data.products);
-
+      setIsLoaded(true);
```

Comment:
This would go nicely in a `.finally` since you also set this below when there's an error.